pages/architecture.md

```diff
@@ -3,22 +3,36 @@ title: Architecture
 weight: 2
 ---
 
+Architecture Overview
+----------------------
 
+Tracers live in your applications and record timing and metadata about
+operations that that took place. They often instrument libraries, so that their
+use is transparent to users. For example, an instrumented web server records
+when it received a request and when it sent a response. The trace data collected
+is called a Span.
 
-Instrumented libraries
-----------------------
+Instrumentation are written to be safe in production and have little overhead.
```

Comment:
Nit: either "Instrumentation is" or "Instrumentations are". In the former case you'd have to also do s/have/has.